client/signup/steps/plans/index.jsx

```diff
@@ -316,6 +319,19 @@ export class PlansStep extends Component {
 		const { hideFreePlan, subHeaderText, translate, eligibleForProPlan, locale } = this.props;
 
 		if ( eligibleForProPlan ) {
+			if ( isStarterPlanEnabled() ) {
+				return hideFreePlan
+					? translate( 'Try risk-free with a 14-day money-back guarantee.' )
+					: translate(
+							'Try risk-free with a 14-day money-back guarantee or {{link}}start with a free site{{/link}}.',
+							{
+								components: {
+									link: <Button onClick={ this.handleFreePlanButtonClick } borderless={ true } />,
+								},
+							}
+					  );
```

Comment:
Need to come back to this. These are two separate strings and not sure if ideal/preferred. I seem to have had trouble getting the final period rendering with something like:

```
				return translate(
					'Try risk-free with a 14-day money-back guarantee{{free}} or {{link}}start with a free site{{/link}}{{/free}}.',
					{
						components: {
							free: hideFreePlan ? null : <span />,
							link: <Button onClick={ this.handleFreePlanButtonClick } borderless={ true } />,
						},
					}
				);
```

guessing `null` not acting friendly 😀